### PR TITLE
Links, Stats, and People Update

### PR DESCRIPTION
**Pull request by AndreasInk**
- Coming soon page for any in development links
- SEO meta tags

### CODE REVIEW

File: src/components/App.tsx [hunk 1]
```diff
@@ -1,18 +1,48 @@
...
-      <Navbar />
-      <Landing />
-      <SectionContainer />
-      <Footer />
+    <BrowserRouter>
+      <Routes>
+        <Route
+          path="/"
+          element={(
+            <div className="App" id="App">
```
Comment (patrickledew) [inline src/components/App.tsx:20]: Duplicate App div

File: src/components/ComingSoon.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import "./comingSoon.scss";
+import Noti from "../res/Noti.svg";
+import Bolt from "../res/Bolt.svg";
+import Back from "../res/Back.svg";
+import Features from "../res/Features.png";
+
+function ComingSoon() {
+  return (
+    <div className="section apps" id="apps">
+      <h1 className="coming-Header">Coming Soon</h1>
```
Comment (patrickledew) [inline src/components/ComingSoon.tsx:10]: The "apps" class and id should only be for the apps section, not the coming soon page

File: src/components/Footer.tsx [hunk 3]
```diff
@@ -76,22 +76,22 @@ function Footer() {
...
           </a>
         </p>
         <p>
           <a
             className="footer-text"
-            href="https://www.craft.do/s/jK9mX5dA3exvRw"
+            href="/ComingSoon"
           >
             Terms and Conditions
           </a>
```
Comment (patrickledew) [inline src/components/Footer.tsx:97]: Use the `Link` component from react-router-dom for navigating to pages within the site

File: src/components/ComingSoon.tsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+      <h3 className="coming-Header">An app that detects infection in realtime</h3>
+      <img alt="Vito Promo" src={Features} />
+
+      <div className="apps-feature">
```
Comment (patrickledew) [inline src/components/ComingSoon.tsx:14]: Again, don't use classes meant for other components

**Review by patrickledew — CHANGES_REQUESTED** [review]
Mostly issues with using styles from different components, some best practice stuff. Still have yet to look at the site but i'll do that when i get back from class lol

**Comment by AndreasInk** [comment]
@patrickledew I made your suggested changes, please let me know if they are sufficient.

**Comment by AndreasInk** [comment]
Merged other commits, so it is now more than new links.

Updates include...
Stats on the project and algorithm 
A code contributors page (People.tsx)
Slight verbiage adjustments
Small UI Improvements

**Review by patrickledew — APPROVED** [review]

**Comment by patrickledew** [comment]
Cool, you're good to merge.

**Pull request merged**